mongodbatlas/atlas_alerts.go

```diff
@@ -0,0 +1,155 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const alertPath = "groups/%s/alerts"
+
+type AlertsService interface {
+	List(context.Context, string, *ListOptions) ([]Alert, *Response, error)
+	GetAnAlert(context.Context, string, string) (*Alert, *Response, error)
+	AckAnAlert(context.Context, string, string, *AlertsRequest) (*Alert, *Response, error)
+}
+
+// AlertServiceOp handles communication with the Alert related methods
+// of the MongoDB Atlas API
+type AlertsServiceOp struct {
+	client *Client
+}
+
+var _ AlertsService = &AlertsServiceOp{}
+
+// Alert represents MongoDB Alert.
+type Alert struct {
+	ID                     string           `json:"id,omitempty"`                     // Unique identifier.
+	GroupID                string           `json:"groupId,omitempty"`                // Unique identifier of the project that owns this alert configuration.
+	AlertConfigID          string           `json:"alertConfigId,omitempty"`          // ID of the alert configuration that triggered this alert.
+	EventTypeName          string           `json:"eventTypeName,omitempty"`          // The type of event that will trigger an alert.
+	Created                string           `json:"created,omitempty"`                // Timestamp in ISO 8601 date and time format in UTC when this alert was opened.
+	Updated                string           `json:"updated,omitempty"`                // Timestamp in ISO 8601 date and time format in UTC when this alert was last updated.
+	Enabled                *bool            `json:"enabled,omitempty"`                // If omitted, the configuration is disabled.
```

Comment:
I'd rather not make this a pointer and not omit it as the behaviour would be the same but I don't have a strong opinion against this

Reply:
The alert_configurations is using a pointer for Enabled. If we decided to change this pointer, I think we should also change the one in alert_configurations.

Reply:
Ok let's leave it as it is for the sake of continuity